requests/src/test/scala/weco/api/requests/fixtures/SierraServiceFixture.scala

```diff
@@ -35,4 +37,75 @@ trait SierraServiceFixture extends HttpFixtures with Akka {
 
       testWith(sierraService)
     }
+
+  def createHoldRequest(
```

Comment:
All this code has been moved unmodified from RequestingScenarioTest, so it can be used in SierraRequestsServiceTest.